next_deployment.md

```diff
@@ -1,6 +1,33 @@
 # Manual steps for next deployment
 
 
-## Configure token lifespan
+### Remove status from Cases not stemming from the `document-review` Workflow
 
-Set `Access Token Lifespan` to 5 minutes in Keycloak.
+Run in Caluma Django shell:
+
+```python
+from caluma.caluma_workflow.models import Case
+
+
+def stats(cases):
+    print(
+        f"Will unset status of {cases.count()} Cases stemming from following Workflows:"
+    )
+    for wf_slug in cases.values_list("workflow__slug", flat=True).distinct():
+        print(f" - {wf_slug}")
+
+
+def do_removal(cases):
+    for case in cases:
+        del case.meta["status"]
+        case.save()
+
+
+cases = Case.objects.exclude(workflow__slug="document-review").filter(
+    meta__status__isnull=False
+)
+
+stats(cases)
+do_removal(cases)
+
+```
```

Comment:
Would it make sense to turn this into a migration? Or at least a management command?

Reply:
I think that's overkill. This is a one-shot thing that basically consists of 3 lines (21-23). The rest is just some fancy reporting whistles.

Reply:
I think that's overkill. This is a one-shot thing that basically consists of 3 lines (21-23). The rest is just some fancy reporting whistles.